README.rst

```diff
@@ -135,8 +135,8 @@ Included localized providers:
 -  `ar\_EG <https://faker.readthedocs.io/en/master/locales/ar_EG.html>`__ - Arabic (Egypt)
 -  `ar\_PS <https://faker.readthedocs.io/en/master/locales/ar_PS.html>`__ - Arabic (Palestine)
 -  `ar\_SA <https://faker.readthedocs.io/en/master/locales/ar_SA.html>`__ - Arabic (Saudi Arabia)
--  `bs\_BA <https://faker.readthedocs.io/en/master/locales/bs_BA.html>`__ - Bosnian
 -  `bg\_BG <https://faker.readthedocs.io/en/master/locales/bg_BG.html>`__ - Bulgarian
+-  `bs\_BA <https://faker.readthedocs.io/en/master/locales/bs_BA.html>`__ - Bosnian
```

Comment:
we keep this sorted alphabetically by the their verbose nome, not their locale code

Reply:
but the list is not ordered by their verbose name either, for example Chinese at the end of the list, Dutch is in the middle.

Reply:
Sorry, my mistake. You're correct :)